modules/ROOT/pages/clauses/limit.adoc

```diff
@@ -59,7 +59,7 @@ d|Rows: 3
 [[limit-subset-rows-using-expression]]
 == Using an expression with `LIMIT` to return a subset of the rows
 
-Limit accepts any expression that evaluates to a positive integer as long as it is not referring to any external variables:
+`LIMIT` accepts any expression that evaluates to a positive integer, as long as it can be statically calculated:
```

Comment:
"statically calculated" should probably be briefly defined - what do you think about either of the below 2 suggestions?

```suggestion
`LIMIT` accepts any expression that evaluates to a positive integer, as long as it can be calculated before the query is run."
```

Alternatively: "...as long as it can be statically calculated (i.e. calculated before the query is run)."